CRM/MembershipExtras/Job/OfflineAutoRenewal/PaymentPlan.php

```diff
@@ -603,13 +603,10 @@ private function createMembership($lineItem, $priceFieldValue) {
    *
    * @param int $membershipID
    *   ID of the membership to be extended.
-   * @param string $startDate
-   *   New start date for the membership.
    */
-  private function extendExistingMembership($membershipID, $startDate) {
+  private function extendExistingMembership($membershipID) {
```

Comment:
Is this OK? I mean, this would re-open MAE-159. Shouldn't we rather fall-back to a default membership status if none of the rules apply on `CRM_MembershipExtras_Hook_Alter_CalculatedMembershipStatus::alterMembershipStatus()`?

Reply:
it does not really, I retested MAE-159 and the period dates on manage installments are just work fine as expected and agreed on in MAE-159, I am also planning to tell sheela to rested MAE-159 anyway.